builder/lxc/step_export.go

```diff
@@ -84,9 +83,37 @@ func (s *stepExport) Run(state multistep.StateBag) multistep.StepAction {
 	commands[0] = []string{
 		"lxc-stop", "--name", name,
 	}
-	commands[1] = []string{
-		"tar", "-C", containerDir, "--numeric-owner", "--anchored", "--exclude=./rootfs/dev/log", "-czf", outputPath, "./rootfs",
+
+	if len(config.ExportConfig.Folders) == 0 {
```

Comment:
would there be case where we want to export rootfs but not call it rootfs.tar.gz ?